bentoml/_internal/external_typing/prometheus.py

```diff
@@ -0,0 +1,100 @@
+from __future__ import annotations
```

Comment:
do we really need this now we have turned on `analysis.useLibraryCodeForTypes = true`?

Reply:
it doesn't work with `analysis.useLibraryCodeForTypes` when the library doesn't even have type hint at the first place :)